test/verify/check-packagekit

```diff
@@ -1196,7 +1196,7 @@ class TestUpdatesSubscriptions(PackageCase):
 
         # download product info from the candlepin machine and install it
         product_file = os.path.join(self.tmpdir, "88888.pem")
-        self.candlepin.download("/home/admin/candlepin/generated_certs/88888.pem", product_file)
+        self.candlepin.download("/home/admin/devel/candlepin/generated_certs/88888.pem", product_file)
```

Comment:
IMHO it would make more sense to adjust services.setup to copy the certificates to the old place. Otherwise you'll have to send similar PRs to our stable branches and sub-man